packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -217,6 +221,15 @@ const styles: ComponentStyles<ISiteManagedClasses, IDevSiteDesignSystem> = {
         alignItems: "center",
         justifyContent: "center"
     },
+    site_transparencyToggleButton: {
+        border: "none",
+        background: "none",
+        height: toPx(32),
+        width: toPx(32),
+        outline: "0",
+        verticalAlign: "middle",
```

Comment:
I'm not sure we still need verticalAlign middle here still. If we do - I think there are better solutions for it.